doc/source/user/quickstart.rst

```diff
@@ -1126,7 +1126,7 @@ indices for each dimension must have the same shape.
     array([[ 2,  6],
            [ 6, 10]])
     >>>
-    >>> a[:, j]                                     # i.e., a[ : , j]
+    >>> a[:, j]                                     # i.e., a[0:4, j]
```

Comment:
I think that'd be more confusing that helpful (and also, shape of `a` is `(3, 4)` so it should've been `0:3`).

Instead, can you please just remove the comment? Slicing and colons are already explained further up in this document, so any form of this `i.e.` isn't helpful.